apps/menu-bar/src/popover/Core.tsx

```diff
@@ -54,14 +58,23 @@ function Core(props: Props) {
     android: undefined,
     ios: undefined,
   });
+  const [userPreferences, setUserPreferences] = useState<UserPreferences>(defaultUserPreferences);
+
+  useEffect(() => {
+    getUserPreferences().then((value) => {
+      saveUserPreferences(value);
+      setUserPreferences(value);
+    });
+  }, []);
```

Comment:
I don't think we need to `saveUserPreferences` here 
```suggestion
  useEffect(() => {
    getUserPreferences().then(setUserPreferences);
  }, []);
```

Reply:
I'll test this again, but I think we actually do, if the user has never set preferences before.

Reply:
The `getUserPreferences()` call now returns a set of default booleans instead of an empty object.